orchagent/intfsorch.cpp

```diff
@@ -1009,6 +1067,12 @@ bool IntfsOrch::addRouterIntfs(sai_object_id_t vrf_id, Port &port)
 
     SWSS_LOG_NOTICE("Create router interface %s MTU %u", port.m_alias.c_str(), port.m_mtu);
 
+    if(gMySwitchType == "voq")
+    {
+        //Sync the interface to add to the SYSTEM_INTERFACE table of CHASSIS_APP_DB
```

Comment:
is it true port.m_type == Port::SYSTEM is the only case this call is needed ?

Reply:
We need to sync the RIF of local port and local LAG to chassis app db. So syncing will be applicable for both PHY and LAG.

Reply:
can we update the comment in that case ?

Reply:
Yes. I'll update the comment.

Reply:
Updated the comment as suggested